R/staggregate.R

```diff
@@ -345,10 +487,22 @@ staggregate_polynomial <- function(data, overlay_weights, daily_agg, time_agg =
     daily_agg = "none"
   }
 
+  # If tolerance supplied is one number, apply to both x and y.
+  # If two numbers, apply first to x and second to y
+  if(length(weights_join_tolerance) == 1){
+    weights_join_tolerance_x <- weights_join_tolerance
+    weights_join_tolerance_y <- weights_join_tolerance
+  } else if(length(weights_join_tolerance == 2)){
+    weights_join_tolerance_x <- weights_join_tolerance[1]
+    weights_join_tolerance_y <- weights_join_tolerance[2]
+  } else{
+    stop(crayon::red("Please provide one digit or a vector of only two digits for weights_join_tolerance."))
+  }
+
   # Aggregate climate data to daily values
   setup_list <- daily_aggregation(data, overlay_weights, daily_agg, time_interval)
 
-  clim_daily <- setup_list[[1]] # Pulls the daily aggregated raster brick
+  clim_daily <- setup_list[[1]] # Pulls the daily aggregated spatRaster stack
```

Comment:
This is actually a comment for a couple of lines down at ` poly_orders <- seq(1:degree)` because `seq()` should have from and to arguments

Reply:
Good catch! I don't think this impacted functionality since seq(x:y) is identical to x:y is identical to seq(x,y), but I've changed it